entrypoint.sh

```diff
@@ -51,7 +51,7 @@ imapfilter_update() {
         kill -TERM "$imapfilter_pid"
         wait "$imapfilter_pid"
     fi
-    imapfilter -c "$config_target" &
+    imapfilter -c "$config_target" -l "$IMAPFILTER_LOGFILE" &
```

Comment:
I like it - but not having a default will cause errors to be displayed.
Please default `IMAPFILTER_LOGFILE` in L13 to `/var/log/imapfilter.log`

Reply:
I added something for an optional parameter in  56c35f3.

We could also do defaulting as below, individually or on L13. 
```suggestion
    imapfilter -c "$config_target" -l "${IMAPFILTER_LOGFILE:/var/log/imapfilter.log}" &
```

Reply:
Yeah, but that'd require to keep the defaults in two different places - something I always try to avoid. I like the implentation you pushed, thanks!